### PR TITLE
add metalctl machine events

### PR DESCRIPTION
**Pull request by mwennrich**
reads system-event-log via `ipmitool sel list`:

```
$ metalctl machine events db56d200-ae8e-11e9-8000-3cecef6b2896 --last 5
8868 | 01/22/2022 | 04:42:09 | Processor | Configuration Error () | Asserted
8869 | 01/22/2022 | 07:24:34 | Processor | Configuration Error () | Asserted
886a | 01/27/2022 | 16:11:53 | Processor | Configuration Error () | Asserted
886b | 01/31/2022 | 01:31:47 | Processor | Configuration Error () | Asserted
886c | 01/31/2022 | 15:07:01 | Processor | Configuration Error () | Asserted
```

### CODE REVIEW

**Comment by majst01** [comment]
Maybe adding a `--limit <count>` option with a sensible default, to show only the last <count> events would help.

**Comment by mwennrich** [comment]
> Maybe adding a `--limit <count>` option with a sensible default, to show only the last events would help.

Added a `--last <count>` with default `10`

File: cmd/machine.go [hunk 1]
```diff
@@ -1424,3 +1438,55 @@ func (c *config) getMachine(args []string) (*models.V1MachineIPMIResponse, error
...
+		fmt.Printf("no ipmi password stored, please specify with --ipmipassword\n")
+	}
+	ipmipassword := viper.GetString("ipmipassword")
+	if ipmipassword != "" {
+		password = ipmipassword
+	}
+
+	args := []string{"-I", intf, "-H", hostAndPort[0], "-p", hostAndPort[1], "-U", usr, "-P", "<hidden>", "sel", "list", viper.GetString("last")}
+	fmt.Printf("connecting to console with:\n%s %s\nExit with ~.\n\n", path, strings.Join(args, " "))
+	args[9] = password
```
Comment (majst01) [inline cmd/machine.go:1487]: this seems dangerous,

**Review by majst01 — APPROVED** [review]

**Comment by Gerrit91** [comment]
I think this is likely to be confused with the machine events entities we have in the metal-api. IMHO this should go beneath `machine ipmi`.

**Comment by majst01** [comment]
> I think this is likely to be confused with the machine events entities we have in the metal-api. IMHO this should go beneath `machine ipmi`.

This is a good point, maybe @mwennrich is willing to change this accordingly.

**Pull request merged**